Bio/SeqFeature.py

```diff
@@ -925,6 +925,25 @@ def __radd__(self, other):
             return self._shift(other)
         else:
             return NotImplemented
+    
+    def __sub__(self,other):
+        """Subtracting an integer will give a new FeatureLocation with
+        its start and end offset shifting back by that amount. For example:
+            
+        >>> print(f1)
+        [5:10]
+        >>> print(f1 + (-100))
+        [-95:-90]
+        >>> print((-200) + f1)
+        [-195:-190]
```

Comment:
These examples are showing the work around of adding negative numbers. You can show ``f1  + (-100)`` does, but should focus on what ``f1 - 100`` does.